torch/_torch_docs.py

```diff
@@ -2619,13 +2619,6 @@ def merge_dicts(*dicts):
     input (Tensor): the PyTorch tensor to test
 """)
 
-add_docstr(torch.is_complex,
```

Comment:
same here.

Also this message is wrong in multiple ways:
- this is about complex, not floating point
- float128 isn't a complex dtype.

Reply:
PR to fix that for master: https://github.com/pytorch/pytorch/pull/35680